test/handlers/admin/data-contributions.js

```diff
@@ -81,12 +81,19 @@ describe('admin data contributions handler', () => {
     });
 
     it('updates the DataContribution and redirects to /admin/data-contributions', () => {
-      const payload = {
-        approved: true,
-        curation_comments: 'My comments',
-      };
+      let payload;
       let dataContributionId;
 
+      factory.create('dataCategory')
+        .then((dataCategory) => {
+          payload = {
+            approved: true,
+            trial_id: '11111111-1111-1111-1111-111111111111',
+            data_category_id: dataCategory.id,
+            curation_comments: 'My comments',
+          };
+        });
```

Comment:
Ah, I see. This sounds a bit familiar, indeed, I guess I've seen some behaviours similar with the one you're describing, in other tests. Good catch!